1.4/Source/VanillaPsycastsExpanded/Paths/Archotechist/Ability_AggressiveHeatDump.cs

```diff
@@ -12,16 +12,21 @@
 
     public class Ability_AggressiveHeatDump : Ability
     {
+        public override float GetRadiusForPawn() =>
+            Mathf.Ceil(Mathf.Min(this.pawn.psychicEntropy.EntropyValue / 20f, 9f * base.GetRadiusForPawn()));
```

Comment:
Could you cap this radius too, just in case?

Reply:
Good point. Breakpoint would be approx 10,000? heat plus 55 radius which is.. probably achievable with some people's cheesy saves.